sphinx/writers/texinfo.py

```diff
@@ -1223,7 +1225,10 @@ def depart_sidebar(self, node: Element) -> None:
         self.depart_topic(node)
 
     def visit_label(self, node: Element) -> None:
-        self.body.append('@w{(')
+        if self.in_footnote:
+            raise nodes.SkipNode
+        else:
+            self.body.append('@w{(')
```

Comment:
Note: This is still needed for citations.

Reply:
All right, so should I add a counter that would increment/decrement once a citation is visited (`in_citation`)?

Reply:
I'm not sure. Let's think that again if we'll need to support.